central/networkgraph/service/service_impl_postgres_benchmark_test.go

```diff
@@ -301,3 +301,81 @@ func BenchmarkGetExternalNetworkFlows(b *testing.B) {
 		},
 	}))
 }
+
+func BenchmarkNetworkGraphExternalFlows(b *testing.B) {
```

Comment:
Consider moving the creation of the data to a separate function from the actual benchmarking of the service. The data creation could actually be split into multiple functions.

Reply:
Done